### DISCUSSION TITLE
Embed Subtitle Preview

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2123] [Q&A]
How do I make embedded subtitles show up during the preview?

**Comment by mifi** [comment]
click the subtitle icon in the lower right of the preivew

<img width="39" alt="Screenshot 2024-08-23 at 23 23 51" src="https://github.com/user-attachments/assets/495d563a-ed84-4350-92b1-385c2cb7da1d">

**Reply by Jay123210599** [reply]
It doesn't have the attachments of the subtitles.
![Screenshot (1056)](https://github.com/user-attachments/assets/72307a39-2f9b-49b2-bd31-7101f9bb72e1)
![sub](https://github.com/user-attachments/assets/adb79baa-6902-44fd-b3c7-a384f1a8196e)

**Reply by mifi** [reply]
Oh you mean styling the subtitles. See #1129

While it seems to be possible to style HTML5 video subtitles and use custom fonts, it seems there's no automated support for this in webvtt files (or at least ffmpeg cannot convert `ass` to `vtt` with this info). There is however  a module that allows rendering `ass` subtitles in HTML5: https://github.com/weizhenye/ASS

also: https://github.com/weizhenye/ass-compiler

**Reply by Jay123210599** [reply]
How do I use this? How will this help?

**Reply by mifi** [reply]
it's not possible for you to use this unless you know how to code. it's more of a future reference for myself or someone else who wants to implement it
